### DISCUSSION TITLE
BOOT_TTY vs TTYDEV vs /dev/tty1

### DISCUSSION THREAD

**Discussion by veremenko-y** [EtchedPixels/FUZIX#1064] [Q&A] [answered]
According to `TTY.rst` and `PORTING`:
`BOOT_TTY` - is strictly for second stage bootloader, but `TTYDEV` is used instead (`Kernel/start.c:298`).
`TTYDEV`- should be used for init but `/dev/tty1` is used instead (`init.c:600`).

Am I missing something or is documentation outdated?

**Comment by EtchedPixels** [comment]
It's all a bit confused and inherited from UZI180. I've been slowly ripping it out.

**Reply by veremenko-y** [reply]
Which one should I trust? Should I trust the code as opposed to the docs?

Option 1: 
* `TTYDEV` - boot
* `/dev/tty1` - init
* `BOOT_TTY` - ignore

Option 2: 
* `BOOT_TTY` - boot
* `TTYDEV` - init
* `/dev/tty1` - ignore

**Comment by EtchedPixels** [comment, marked as answer]
Code

(well nothing stops kputchar writing to usb and serial)
